Computer History Museum Archive.js

```diff
@@ -0,0 +1,988 @@
+				"archive": "Computer History Museum",
+				"archiveLocation": "X7120.2014",
+				"extra": "Catalog Number: 102746874\nType: Document\nCategory: Transcription\nSubject: Xbox; Xbox 360; Xbox One; Sony Corporation; Microsoft Corporation; Playstation; Windows; International Business Machines Corporation (IBM); ATI Technologies Inc.; Central Processing Unit (CPU); Graphics Processing Unit (GPU); Nvidia Corporation; Media Center PC (personal computer); Halo; Kinect; Performance Optimization With Enhanced RISC--Performance Computing (PowerPC); Flextronics; Wistron; Allard, J; Bach, Robbie\nCollection Title: Oral history collection\nCredit: Computer History Museum",
+				"libraryCatalog": "Computer History Museum Archive",
+				"numPages": "27",
+				"place": "Mountain View, California",
+				"publisher": "Computer History Museum",
+				"rights": "Computer History Museum",
+				"attachments": [
+					{
+						"mimeType": "application/pdf",
+						"title": "102746874-05-01-acc.pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/search/?s=oral+history&page=4",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102738261",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Su, Stephen oral history",
+				"creators": [
+					{
+						"firstName": "Douglas",
+						"lastName": "Fairbairn",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Stephen",
+						"lastName": "Su",
+						"creatorType": "contributor"
+					}
+				],
+				"date": "2017-04-19",
+				"abstractNote": "Stephen Su grew up on Taiwan, but in 1980 he came to the US to attend high school and college, including studying semiconductors at Caltech. He worked for Motorola for a period of time before returning to school in 1992 to get an MBA from Kellog. Upon graduation, he joined Boston Consulting Group and went to Hong Kong on a consulting assignment. \nIn 1998, Stephen returned to Taiwan, working for Primax.  While there he spent several years managing the Mobile Accessories group, responsible for developing accessories for mobile phone makers like Nokia and Apple.  In particular, he was responsible for developing the camera module for several generations of Apple’s iPhone. He tells many interesting stories about working with Apple on this very important program. \nIn 2009, he was recruited to join ITRI, where he is involved with helping steer Taiwan into lucrative new markets through careful investments in promising new technologies.",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X8201.2017",
+				"extra": "Catalog Number: 102738261\nType: Moving image\nFormat: MOV\nCategory: Oral history\nSubject: ITRI; Apple; iPhone; Camera\nCredit: Computer History Museum",
```

Comment:
`Subject` should become tags.

Reply:
And the `Video Recording` item type, which this should be, has a `Format` field (though that doesn't add a lot of value).

Reply:
fixed.